charts/kubewarden-controller/chart-values.yaml

```diff
@@ -46,7 +46,7 @@ image:
   # controller image to be used
   repository: "kubewarden/kubewarden-controller"
   # image tag
-  tag: v1.7.0-rc2
+  tag: v1.7.0-rc3
   pullPolicy: IfNotPresent
 preDeleteJob:
   image:
```

Comment:
`.Values.preDeleteJob.image.tag` should be `v1.27.6` from https://github.com/kubewarden/rancher-kubectl-builder/pkgs/container/kubectl.

(we don't have that automated).

Reply:
Afaics, I cannot change that line here in the PR. Therefore, I'll fix it upstream and the PR can be rebased it on top of that change.

Reply:
https://github.com/kubewarden/helm-charts/pull/290